### PR TITLE
DOMA-3771 sms after ticket creation

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
If when creating a ticket for a resident, a `resident` was not found by the entered contact number, then we send an SMS to this number

**TODO:**
- [x] pass `organization` in `sendMessage` and `Message.create` - [pull request](https://github.com/open-condo-software/condo/pull/2055)
- [x] add `BlackList` model in `notification` domain - [pull request](https://github.com/open-condo-software/condo/pull/2064)

### CODE REVIEW

File: apps/condo/domains/notification/constants/constants.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const CUSTOMER_IMPORTANT_NOTE_TYPE = 'CUSTOMER_IMPORTANT_NOTE_TYPE'
 const MESSAGE_FORWARDED_TO_SUPPORT_TYPE = 'MESSAGE_FORWARDED_TO_SUPPORT'
 const TICKET_ASSIGNEE_CONNECTED_TYPE = 'TICKET_ASSIGNEE_CONNECTED'
 const TICKET_EXECUTOR_CONNECTED_TYPE = 'TICKET_EXECUTOR_CONNECTED'
+const TICKET_WITHOUT_RESIDENT_CREATED_TYPE = 'TICKET_WITHOUT_RESIDENT_CREATED'
```
Comment (akarjakin) [inline apps/condo/domains/notification/constants/constants.js:26]: TRACK_TICKET_IN_DOMA_APP_TYPE

**Review by akarjakin — APPROVED** [review]

**Review by AntonAL — APPROVED** [review]

File: apps/condo/domains/notification/schema/SendMessageService.js [hunk 2]
```diff
@@ -134,11 +135,22 @@ const SendMessageService = new GQLCustomSchema('SendMessageService', {
                 if (to.phone) messageAttrs.phone = to.phone
                 if (to.user) messageAttrs.user = { connect: to.user }
 
-                const message = await Message.create(context, messageAttrs)
+                let messageWithSameUniqKey
+                if (uniqKey) {
+                    messageWithSameUniqKey = await getByCondition('Message', {
```
Comment (pahaz) [inline apps/condo/domains/notification/schema/SendMessageService.js:140]: DeletetAt

File: apps/condo/lang/ru/messages/TRACK_TICKET_IN_DOMA_APP/default.njk [hunk 3]
```diff
@@ -0,0 +1 @@
+Следите за вашей заявкой в приложении Doma: http://onelink.to/doma
```
Comment (pahaz) [inline apps/condo/lang/ru/messages/TRACK_TICKET_IN_DOMA_APP/default.njk:1]: It’s better to remove hard coded links, but probably no in this pr. It looks like another task

File: apps/condo/domains/notification/schema/SendMessageService.js [hunk 4]
```diff
@@ -134,11 +135,22 @@ const SendMessageService = new GQLCustomSchema('SendMessageService', {
                 if (to.phone) messageAttrs.phone = to.phone
                 if (to.user) messageAttrs.user = { connect: to.user }
 
-                const message = await Message.create(context, messageAttrs)
+                let messageWithSameUniqKey
+                if (uniqKey) {
```
Comment (pahaz) [inline apps/condo/domains/notification/schema/SendMessageService.js:139]: It’s better to remove try/catch in others places

**Review by pahaz — APPROVED** [review]

**Review by AntonAL — APPROVED** [review]

**Pull request merged**